features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -397,6 +397,11 @@
         <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.onkyo/${project.version}</bundle>
     </feature>
 
+    <feature name="openhab-binding-opendaikin" description="Daikin A/C Unit Binding" version="${project.version}">
```

Comment:
It's easier to find the add-on in the list when the description matches the bindingId and name used everywhere else.

```suggestion
    <feature name="openhab-binding-opendaikin" description="OpenDaikin Binding" version="${project.version}">
```